data_collections/getEvents.py

```
@@ -0,0 +1,85 @@
+            "Type": "Event",
+            "Title": title,
+            "Description": descrip,
+            "whenDate": when,
+            "pubDate": entry.get("published", ""),
+            "Location": location,
+            "link": entry.get("link", ""),
+            "entryDate": datetime.datetime.now(),
+        }
+
+        events.append(event)  # list of each event which is stored in a dictionary
+    print
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)